src/gtpV2Codec/ieClasses/arpIe.cpp

```diff
@@ -68,6 +74,13 @@ bool ArpIe::decodeArpIe(MsgBuffer &buffer, ArpIeData &data, Uint16 length)
         errorStream.add("Attempt to read beyond IE boundary: pl\n");
         return false;
     }
+    buffer.skipBits(1);
+    // confirm that we are not reading beyond the IE boundary
+    if (buffer.getCurrentIndex() > ieBoundary)
+    {
+        errorStream.add("Attempt to read beyond IE boundary: pl\n");
+        return false;
+    }
```

Comment:
@pingping-lin  this might be good to have fix in Openmme/paging but not in Nucleus.

Reply:
Wanted to know what was side effect of this defect in the MME ?

Reply:
agree, in Nucleus, it is better to fix in the autogeneration.
There is a judgement after mme parse all the IEs, whether there are still data left. If so mme will treat it as a failure. Because it missed to parse 1 bit before pvi, so yes there are still 1 bit data left after parse all IEs, and mme thought it failed and did not do further work like send ddn ack and did not trigger the paging packet.